src/pages/SearchPage.js

```diff
@@ -0,0 +1,123 @@
+import styled from 'styled-components';
+import Header from '../components/Header';
+import SearchResultCard from '../components/SearchResultCard';
+import Navigation from '../components/Navigation';
+import ScreenReaderOnly from '../components/ScreenReaderOnly';
+import LoadingSpinner from '../components/LoadingSpinner';
+import { ReactComponent as SearchIcon } from '../assets/icons/search_icon.svg';
+
+import useSearch from '../hooks/useSearch';
+
+export default function SearchPage() {
+  const { isLoading, results, handleSearch } = useSearch();
+
+  return (
+    <Wrapper>
+      <Header />
+      <SearchWrapper>
+        <label htmlFor="search">
+          <ScreenReaderOnly>Suche</ScreenReaderOnly>
+        </label>
+        <input
+          id="search"
+          name="search"
+          placeholder="Star Wars, Suits,..."
+          type="text"
+          onChange={event => handleSearch(event)}
+          onKeyDown={event =>
+            (event.key === 'Enter' || event.key === 'Escape') &&
+            event.target.blur()
+          }
+          required
+        />
+      </SearchWrapper>
+      {isLoading && <LoadingSpinner />}
+      {results.length !== 0 ? (
+        <StyledList>
+          {results
+            .filter(result => result.media_type !== 'person')
+            .map(
+              ({
+                id,
+                name,
+                title,
+                poster_path,
+                release_date,
+                first_air_date,
```

Comment:
Looks really interesting, didnt see it before like this , didnt even know that it works like this as well 🤩